driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java

```diff
@@ -238,6 +249,10 @@ private OIndexManager getIndexManager() {
 
     @Override
     public Iterator<Edge> edges(Object... edgeIds) {
+        Set<Class<?>> edgeIdTypes = Stream.of(edgeIds).map(edgeId -> edgeId.getClass()).collect(toSet());
+        if(edgeIdTypes.size() > 1 && edgeIdTypes.contains(OrientEdge.class))
+            throw Graph.Exceptions.idArgsMustBeEitherIdOrElement();
```

Comment:
only by if we disable features.... but that would also turn off a huge set of tests...

well, I can try to be more selective them that.... HACKER MODE ON =D